src/resolvers/Mutation/inviteShopMember.js

```diff
@@ -17,12 +19,24 @@ import { decodeGroupOpaqueId, decodeShopOpaqueId } from "../../xforms/id.js";
  */
 export default async function inviteShopMember(_, { input }, context) {
   const { email, groupId, name, shopId, clientMutationId = null } = input;
-  const decodedGroupId = decodeGroupOpaqueId(groupId);
+  let { groupIds } = input;
+
+  // If user is using deprecated `groupId` instead of `groupIds`, populate `groupIds`
+  if (groupId && (!Array.isArray(groupIds) || groupIds.length === 0)) {
+    groupIds = [groupId];
+  }
+
+  // If user is passing both `groupId` and `groupIds`, throw an error
+  if (groupId && Array.isArray(groupIds) && groupIds.length > 0) {
+    throw new ReactionError("invalid-parameter", "Can't specify both groupId and groupIds.");
+  }
```

Comment:
This check would have to come before the previous `if`. Otherwise it will always fail because you just set `groupIds` to `[groupId]` above.